packages/react-router-website/api/BrowserPrompt.md

```diff
@@ -0,0 +1,25 @@
+# BrowserPrompt
```

Comment:
Yes, I didn't want to duplicate docs already within the Prompt. If we update Prompt we would have to remember to keep this up to date. That;s why I state that this is a wrapper to Prompt.